README.md

```diff
@@ -0,0 +1,40 @@
+# Ô Chap'
+
+Ô Chap' is an e-commerce website for selling all kinds of hats!
+
+## Installation
+
+### Requirements
+
+- Git
+- JDK 1.8 or later
+- Maven 3.2 or later
+
+### Instructions
+
+```bash
+# Go to the directory where you want to clone the project.
+cd path/where/to/put/the/project
+
+# Clone the project.
+git clone https://github.com/AMT-2021/AMT
+
+# Run the project locally with Maven.
+mvn spring-boot:run
+```
+
+You should be able to access the website on [http://localhost:8080/](http://localhost:8080/).
+
+## Usage
+
+Once the project is running locally, you can access it with a web browser on [http://localhost:8080/](http://localhost:8080/).
+
+## Contributing
+Pull requests are welcome. For major changes, please open an issue first to discuss what you would like to change.
```

Comment:
What is a major change? An example of something more precise would be something along the lines of: _pull requests for bugfixes are very welcome. For new features or other changes, please open an issue first._